changelog-2023.04.02-7.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/packaging/app.organicmaps.desktop.metainfo.xml b/packaging/app.organicmaps.desktop.metainfo.xml
+index 942a5bc9db..0612a5ab55 100644
+--- a/packaging/app.organicmaps.desktop.metainfo.xml
++++ b/packaging/app.organicmaps.desktop.metainfo.xml
+@@ -113,6 +113,39 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2023.04.02-7" date="2023-04-02">
++     <description>
++       <p>Highlights:</p>
++         <ul>
++           <li>New OpenStreetMap data as of March 29</li>
++           <li>Correctly import KML bookmarks with track data</li>
++           <li>Allow adding music and musical_instruments in the Editor</li>
++           <li>Sewing shops and haberdasheries are searchable</li>
++         </ul>
++       <p>Map Styles:</p>
++         <ul>
++           <li>Distinguish supermarkets and convenience stores</li>
++           <li>New shop types: agrarian, appliance, bed, outpost, gas, key cutter, locksmith, carpet, motorcycle repair, pet grooming, hi-fi, bathroom furnishing, curtain, fashion accessories, cannabis, herbalist</li>
++           <li>Added money transfer, bicycle charging stations, power plant</li>
++           <li>Made captions mandatory for capitals with a star icon</li>
++           <li>Made park, forest, and scrub colors different</li>
++           <li>Added metal construction icons</li>
++           <li>Improved aerial ways, cable cars, power lines, fences</li>
++           <li>Reduce Barcelona subway visibility</li>
++           <li>Fixed isolines that are drawn over buildings, roads, etc.</li>
++           <li>Fixed some POI captions displayed earlier than their icons</li>
++         </ul>
++       <p>Translations:</p>
++         <ul>
++           <li>Improved German, Turkish, Belarusian, Hungarian translations</li>
++         </ul>
++       <p>Linux:</p>
```

Comment:
I wouldn't say it's Linux only. It's also for mac, and it would work for Windows too.

Reply:
The flatpak package format is Linux only, and the appstream metadata that we are patching here is Linux only.
IMHO, relevant changes here are those that:
1. Pertaining to the common codebase and therefore to all the platform
2. Those that are pertaining to Linux.

Why would Linux users find some extra noise about mac or windows relevant? And then why not about iOS and so on?
The way I see, the Apple App store version history, or the F-droid history doesn't generate noise about other platforms either. To me it that makes sense to follow that pattern here.

Reply:
My comment was not about mentioning other platforms. It was about not highlighting Linux. Because ALL changes in the changelog are done for Linux and also for Mac. There is no need to mention explicitly "Linux" here. Otherwise, it looks like all changes before "Linux" section were done for all platforms, but "Linux" changes were done only for Linux. It is not true.

Reply:
All right. I have dropped the last paragraph and moved the items into the `Highlights` paragraphs.